File: src/sdk-reference/php/3/memory-storage/zunionstore/snippets/zunionstore-1.js

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $count = $kuzzle->memoryStorage()->zunionstore('destination', ['key1', 'key2', '...']);
}
catch (ErrorException $e) {

}

File: .repos/sdk/sdk-android-3/doc/3/core-classes/memory-storage/zunionstore/snippets/zunionstore-1.java

String[] keys = new String[]{"key1", "key2", "..."};

kuzzle.memoryStorage.zunionstore("destination", keys, new ResponseListener<Long>() {
  @Override
  public void onSuccess(int count) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
